src/src/main/java/org/redfrog404/mobycraft/entity/EntityChaosMonkey.java

```diff
@@ -165,7 +161,7 @@ public void onUpdate() {
 		}
 
 		Container container = factory.getListCommands().getWithName(name);
-		Mobycraft.getMainCommand().getDockerClient()
+		MainCommand.getDockerClient()
```

Comment:
Over in Mobycraft.java you've got this to get MainCommand:
     public static MainCommand getMainCommand() {
		return commands;
	}
but MainCommand itself already has getDocherClient() as a static method, so I'm thinking you might just want to use that rather than fetching it via Mobycraft:
public static DockerClient getDockerClient() {...}

Note: As with all other mods suggested on this pull request, feel free not to accept the suggestion if you prefer the existing approach.